packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -48,8 +48,8 @@ const styles = StyleSheet.create({
     borderStyle: 'solid',
   },
   image: {
-    width: 20,
-    height: 20,
+    width: 120,
```

Comment:
Would this break other accessibility examples that use `styles.image`? Could create a separate `styles.disabledImage`? Did you want me to add? Otherwise I think this looks good!

Reply:
@lunaleaps thanks a lot for the codereview.. sorry for this. I just updated the code with commit https://github.com/facebook/react-native/pull/31252/commits/f5e4a162db917866b9213d25205715093b02a092 I remain available :pray: